Tests/test_Uniprot.py

```diff
@@ -85,180 +132,289 @@ def test_uni003(self):
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_miscellaneous"],
+            ["Present with 20200 molecules/cell in log phase SD medium."],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_onlineinformation"],
+            ["NIEHS-SNPs@http://egp.gs.washington.edu/data/api5/"],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_pathway"],
+            [
+                "Lipid metabolism; malonyl-CoA biosynthesis; "
+                "malonyl-CoA from acetyl-CoA: step 1/1."
+            ],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_RNAediting"],
+            [
+                "Partially edited. RNA editing generates receptor "
+                "isoforms that differ in their ability to interact "
+                "with the phospholipase C signaling cascade in a "
+                "transfected cell line, suggesting that this RNA "
+                "processing event may contribute to the modulation "
+                "of serotonergic neurotransmission in the central "
+                "nervous system."
+            ],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_PTM"],
+            ["Acetylation at Lys-251 impairs antiapoptotic function."],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_pharmaceutical"],
+            [
+                "Could be used as a possible therapeutic agent for "
+                "treating rheumatoid arthritis."
+            ],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_polymorphism"],
+            [
+                "Position 23 is polymorphic; the frequencies in "
+                "unrelated Caucasians are 0.87 for Cys and 0.13 "
+                "for Ser."
+            ],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_similarity"],
+            ["Belongs to the FARP (FMRFamide related peptide) family."],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_subcellularlocation_location"], ["Secreted"]
+        )
+        self.assertEqual(seq_record.annotations["comment_subunit"], ["Homodimer."])
+        self.assertEqual(
+            seq_record.annotations["comment_tissuespecificity"],
+            [
+                "Each flp gene is expressed in a distinct set of "
+                "neurons. Flp-13 is expressed in the ASE sensory "
+                "neurons, the DD motor neurons, the 15, M3 and M5 "
+                "cholinergic pharyngeal motoneurons, and the ASG, "
+                "ASK and BAG neurons."
+            ],
+        )
+        self.assertEqual(
+            seq_record.annotations["comment_toxicdose"],
+            [
+                "LD(50) is 50 ug/kg in mouse by "
+                "intracerebroventricular injection "
+                "and 600 ng/g in Blatella germanica."
```

Comment:
These three lines could fit on two by doing the breaks differently. Same applies to lots of the examples above - but to be honest, it would be a lot of manual effort for a very slight improvement. Your effort would be more valuable elsewhere.